xml/System/ReadOnlySpan`1.xml

```diff
@@ -84,10 +90,16 @@
         <Parameter Name="length" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="pointer">To be added.</param>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="pointer">A pointer to the starting address of a specified number of bytes in memory.</param>
+        <param name="length">The number of bytes to be included.</param>
```

Comment:
This is wrong.  It's not the number of bytes, it's the number of Ts:
https://github.com/dotnet/coreclr/blob/af0b442c17fe3a9a95370e506a36ce72ee99ea80/src/System.Private.CoreLib/shared/System/ReadOnlySpan.Fast.cs#L92